core/src/main/resources/hudson/model/Label/index.jelly

```diff
@@ -39,12 +47,12 @@ THE SOFTWARE.
         <st:include page="summary.jelly" from="${a}" optional="true" it="${a}" />
       </j:forEach>
 
-      <div>
-        <h2>${%Nodes}</h2>
+      <div style="padding-top:10px">
+        <h2>${%Nodes} (${it.sortedNodes.size()})</h2>
         <j:forEach var="n" items="${it.sortedNodes}">
           <j:set var="c" value="${app.getComputer(n.nodeName)}"/>
           <j:set var="url" value="${rootURL}/${c.url}"/>
-          <nobr>
+          <nobr tooltip="${n.nodeDescription}">
```

Comment:
XSS vulnerability by attackers able to configure node labels (unconfirmed)

Reply:
thx, I forgot to use app.markupFormatter.translate()

Reply:
It would be incorrect to do that before #6511. You probably want `h.htmlAttributeEscape`.